Text change

flopp999 opened the pull request:
**What this PR does / why we need it:**
I found some inconsistent search text

**Which issue(s) this PR fixes:**
https://github.com/grafana/grafana/issues/22934

**Fixes #**
made all, I hope, search fields more consistent

flopp999 committed:
Text change

made it more understandable that this is a search field

aknuds1 commented:
Can you please rewrite the PR using our standard template and with a descriptive title? Thanks!

flopp999 committed:
Made "New Folder"-button green

Since New Dashboard was green I thought New Folder; Import also would be green. Green seems to be the color for creating new items

flopp999 committed:
Update playlist_search.html

flopp999 committed:
change search text

flopp999 committed:
wrong link

flopp999 committed:
Update playlist.md

flopp999 committed:
Update manage_dashboards.html

flopp999 committed:
Update manage_dashboards.html

flopp999 committed:
replaced Filter

flopp999 committed:
replaced Filter

flopp999 committed:
replaced Filter

flopp999 committed:
replaced filter

flopp999 committed:
changed text for placeholder

flopp999 committed:
Update UsersActionBar.tsx

flopp999 committed:
Update UserListAdminPage.tsx

flopp999 commented:
I don't know if I by mistake as for review from @oddlittlebird or it was the system. :)

If it was me, can you please remove the request?

torkelo reviewed (approved):
This looks good, and fixes a broken docs link!

aknuds1 commented:
The PR breaks front-end tests apparently.

oddlittlebird commented:
@aknuds1, could you look into why the test is broken? Fixing the link shouldn't do it. And if the link was bad, then it would be failing on a different test.

aknuds1 commented:
@oddlittlebird The PR does a lot more than changing documentation, it also changes front-end code. That must be why the tests fail.

aknuds1 commented:
It's assigned to the front-end squad, so hopefully they'll have a look.

oddlittlebird commented:
AH, gotcha.


On Mon, Mar 23, 2020 at 9:16 AM Arve Knudsen <notifications@github.com>
wrote:

> @oddlittlebird <https://github.com/oddlittlebird> The PR does a lot more
> than changing documentation, it also changes front-end code. That must be
> why the tests fail.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/grafana/grafana/pull/22929#issuecomment-602703167>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMNF6KJ2GYR3TIKZUZBPJULRI6DMXANCNFSM4LQOMJGA>
> .
>


-- 
Diana Payton (She/Her)
Sr. Technical Writer at Grafana Labs

dprokop committed:
update snapshots

dprokop committed:
Merge branch 'master' into patch-13
